body-segmentation/src/selfie_segmentation_mediapipe/types.ts

```diff
@@ -47,6 +47,7 @@ export interface MediaPipeSelfieSegmentationMediaPipeModelConfig extends
     MediaPipeSelfieSegmentationModelConfig {
   runtime: 'mediapipe';
   solutionPath?: string;
+  locateFile?: (path: string, base: string) => string;
```

Comment:
Please add a comment to the interface describing this option, and similarly in README.md in the same folder as this file.